Testreglar/1.4.4/Nett/nett-1.4.4a.json

```diff
@@ -0,0 +1,263 @@
+{
+	"namn": "Nett-1.4.4a Tekst kan forstørrast til minst 200% uten tap av innhold og funksjonalitet 2023",
+	"id": "nett-1.4.4a",
+	"testlabId": 451,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nn",
+	"kravTilSamsvar": "<ul>\r\n<li>All tekst kan forstørres opp til minst 200 % visning uten at innhold eller funksjonalitet går tapt</li>\r\n<li>Dersom forstørring tilbys via en mekanisme på den aktuelle nettsiden, skal den være lett å finne</li>\r\n</ul>",
+	"side": "2.1",
+	"element": "Side",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "3.1"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		},
+		{
+			"title": "3.8"
+		},
+		{
+			"title": "3.9"
+		},
+		{
+			"title": "3.10"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Kva side testar du?",
+			"ht": "<p>Oppgi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har testsida tekst?",
+			"ht": "<p>Tekst som i følgje kravet er unntatt:</p>\n<ul>\n<li>teksting i video</li>\n<li>bilde av tekst</li>\n</ul>\n<p><strong>Merk</strong>: Du skal ikkje teste vidare på nettsider utan tekst.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Nettsida har ikkje tekst."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Sett oppløysinga på skjermen du testar med til 1920 x 1080 pikslar",
+			"ht": "<p>Viss du ikkje har dette som standard, må du endre på oppløysinga, før du startar test. </p>\n<p>Slik endrer du innstilling: System &gt;Skjerm &gt;Skjermoppløsning &gt;1920x1080 &gt;Behold Endringer</p>\n<p><strong>Merk</strong>: Oppløysinga vi testar med, byggjer på statistikk over dei mest brukte skjermoppløysingane i verda per 2022-2023.</p>",
+			"type": "instruksjon",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Har nettlesaren ein funksjon for å zoome til minst 200 %?",
```

Comment:
Litt usikker på om vi trenger dette, eller vi heller skal kreve at ****testen skal gjennomføres i en nettleser** som har zoom-funksjon.**  Det er litt "søkt" å holde en webredaktør ansvarlig for at noen bruker en eller annen bisarr nettleser som ikke har zoom funksjonalitet.